src/serialization/SerializationService.ts

```diff
@@ -291,44 +325,36 @@ export class SerializationServiceV1 implements SerializationService {
     }
 
     private registerDefaultSerializers(): void {
-        this.registerSerializer('string', new StringSerializer());
-        this.registerSerializer('double', new DoubleSerializer());
-        this.registerSerializer('byte', new ByteSerializer());
-        this.registerSerializer('boolean', new BooleanSerializer());
-        this.registerSerializer('null', new NullSerializer());
-        this.registerSerializer('short', new ShortSerializer());
-        this.registerSerializer('integer', new IntegerSerializer());
-        this.registerSerializer('long', new LongSerializer());
-        this.registerSerializer('float', new FloatSerializer());
-        this.registerSerializer('char', new CharSerializer());
-        this.registerSerializer('date', new DateSerializer());
-        this.registerSerializer('localDate', new LocalDateSerializer());
-        this.registerSerializer('localTime', new LocalTimeSerializer());
-        this.registerSerializer('localDateTime', new LocalDateTimeSerializer());
-        this.registerSerializer('offsetDateTime', new OffsetDateTimeSerializer());
-        this.registerSerializer('byteArray', new ByteArraySerializer());
-        this.registerSerializer('charArray', new CharArraySerializer());
-        this.registerSerializer('booleanArray', new BooleanArraySerializer());
-        this.registerSerializer('shortArray', new ShortArraySerializer());
-        this.registerSerializer('integerArray', new IntegerArraySerializer());
-        this.registerSerializer('longArray', new LongArraySerializer());
-        this.registerSerializer('doubleArray', new DoubleArraySerializer());
-        this.registerSerializer('stringArray', new StringArraySerializer());
-        this.registerSerializer('javaClass', new JavaClassSerializer());
-        this.registerSerializer('floatArray', new FloatArraySerializer());
-        this.registerSerializer('arrayList', new ArrayListSerializer());
-        this.registerSerializer('linkedList', new LinkedListSerializer());
-        this.registerSerializer('uuid', new UuidSerializer());
-        this.registerSerializer('bigDecimal', new BigDecimalSerializer());
-        this.registerSerializer('bigint', new BigIntSerializer());
-        this.registerSerializer('javaArray', new JavaArraySerializer());
-        this.registerSerializer('!compact', this.compactStreamSerializer);
-        this.registerSerializer('identified', this.identifiedSerializer);
-        this.registerSerializer('!portable', this.portableSerializer);
+        this.registerSerializer(String.prototype.constructor, new StringSerializer(), new StringArraySerializer());
```

Comment:
```suggestion
        this.registerSerializer(String, new StringSerializer(), new StringArraySerializer());
```

Similar for others, they are the same object. 
```
> String.prototype.constructor === String
true
> 
```